go/libraries/doltcore/sqle/dtablefunctions/dolt_log.go

```diff
@@ -515,41 +530,34 @@ func (ltf *LogTableFunction) RowIter(ctx *sql.Context, row sql.Row) (sql.RowIter
 	return ltf.NewDotDotLogTableFunctionRowIter(ctx, sqledb.DbData().Ddb, commits, notCommits, matchFunc, cHashToRefs, ltf.tableNames)
 }
 
-// evaluateArguments returns revisionValStrs, notRevisionValStrs, and three dot boolean.
-// It evaluates the argument expressions to turn them into values this LogTableFunction
-// can use. Note that this method only evals the expressions, and doesn't validate the values.
-func (ltf *LogTableFunction) evaluateArguments() (revisionValStrs []string, notRevisionValStrs []string, threeDot bool, err error) {
-	for _, expr := range ltf.revisionExprs {
-		valStr, err := expressionToString(ltf.ctx, expr)
-		if err != nil {
-			return nil, nil, false, err
-		}
-
-		if strings.Contains(valStr, "..") {
-			if strings.Contains(valStr, "...") {
-				refs := strings.Split(valStr, "...")
-				return refs, nil, true, nil
-			}
-			refs := strings.Split(valStr, "..")
-			return []string{refs[1]}, []string{refs[0]}, false, nil
+// parseRevisionRange processes a revision string for range syntax (.., ...)
+// Returns (revisions, notRevisions, threeDot, isRange)
+func parseRevisionRange(revisionStr string) ([]string, []string, bool, bool) {
+	if strings.Contains(revisionStr, "..") {
+		if strings.Contains(revisionStr, "...") {
+			refs := strings.Split(revisionStr, "...")
+			return refs, nil, true, true
 		}
-
-		revisionValStrs = append(revisionValStrs, valStr)
+		refs := strings.Split(revisionStr, "..")
+		return []string{refs[1]}, []string{refs[0]}, false, true
 	}
+	return nil, nil, false, false
+}
 
-	for _, notExpr := range ltf.notRevisionExprs {
-		notValStr, err := expressionToString(ltf.ctx, notExpr)
-		if err != nil {
-			return nil, nil, false, err
-		}
-
-		if strings.HasPrefix(notValStr, "^") {
-			notValStr = strings.TrimPrefix(notValStr, "^")
+// evaluateArguments returns revisionValStrs, notRevisionValStrs, and three dot boolean.
+// It processes the stored revision strings from the CLI parser and handles range syntax.
+func (ltf *LogTableFunction) evaluateArguments() (revisionValStrs []string, notRevisionValStrs []string, threeDot bool, err error) {
+	// Process stored revision strings (from CLI parser)
+	for _, revisionStr := range ltf.revisionStrs {
+		if revs, notRevs, isThreeDot, isRange := parseRevisionRange(revisionStr); isRange {
+			return revs, notRevs, isThreeDot, nil
 		}
-
-		notRevisionValStrs = append(notRevisionValStrs, notValStr)
+		revisionValStrs = append(revisionValStrs, revisionStr)
 	}
 
+	// Process stored not-revision strings
+	notRevisionValStrs = append(notRevisionValStrs, ltf.notRevisionStrs...)
```

Comment:
This seems a little fishy, too... there's not any "processing" actually going on here, right? We're just making a copy of `ltf.notRevisionStrs`. This feels like we can simplify more here. 

Speaking of simplifying... the `error` return param is never actually used here. Sometimes it's nice to leave room for that in an interface, but for a helper function like this, you should just simplify and not force the caller to check an error if one isn't ever going to be returned. 

Take another look at this code and see how much more you can simplify here.